source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -48,13 +48,16 @@ export default class NetworkStatusStore extends Store {
   @observable _localDifficultyStartedWith = null;
 
   _timeDifferencePollInterval: ?number = null;
+  _pollerBlocked: ?boolean = false;
```

Comment:
@tomothespian `?boolean` is not needed here. Maybe we don't even need type - just `false` should be enough.